common/core/gpudevice.cpp

```diff
@@ -1022,8 +1022,6 @@ void GpuDevice::HandleRoutine() {
   //    we need to take control.
   // TODO: Add splash screen support.
   if (lock_fd_ != -1) {
-    display_manager_->IgnoreUpdates();
```

Comment:
looks good to me. 
but why double setting FrameStateTracker::kIgnoreUpdates and FrameStateTracker::kForceIgnoreUpdates will cause mosaic and logical mode issue?

Reply:
We just change the status 'forceignore' here. and this flag only work in drm and physical display. it is a bug for ignoring and forcerefreshing on Mosaic and logic display. the flag 'forceignore' didn't work for them. It should be fixed. I close this one. and will prepare another for adding this flag into mosaic and logical too